plugins/logitech-bulkcontroller/README.md

```diff
@@ -27,13 +27,59 @@ These devices use the standard USB DeviceInstanceId values, e.g.
 
 This plugin uses the following plugin-specific quirks:
 
+### `Flags=check-buffer-size`
```

Comment:
Please include quirk file, if modified

Reply:
I've left this disabled for now, until we work out the 16kb bug-- we can add it per-device when we know what devices and what versions work.